lib/elixir/unicode/tokenizer.ex

```diff
@@ -1,6 +1,29 @@
 defmodule String.Tokenizer do
   @moduledoc false
 
+  ## Custom normalization definitions
+  #
+  # These codepoints will be normalized from => to, and their
+  # scriptset will be the union of both. If one of the two
+  # codepoints is Script='Common|Inherited', this means both
+  # codepoints can be used without unsafe script mixing;
+  # similarly, they are exempted from the Restricted list.
+  #
+  elixir_normalizations = %{
+    # NFKC-based normalizations for codepoints in use
+    'µ' => 'μ',
+
+    # math symbols can be supported by this same means;
+    # this 'easter egg', which no users are likely to rely on,
+    # is put here so we can exercise the mechanism for allowing
+    # mathematical symbols, where perhaps even the 'to'
+    # codepoint will be Script=Common.
+    'ℕ' => 'ℕ',
+    '𝕩' => '𝕩'
```

Comment:
Let's remove the easter egg for now, but we can keep the code and leave the actual test commented out.